src/assets/config_files/usecases.json

```diff
@@ -655,6 +655,149 @@
             "justcopy": true
           }
         ]
+      },
+	  {
+        "title": "How to setup a custom axon (BluePyOpt) and simulate with it a regular firing granule cell (2019)",
+        "description": "The purpose of this usecase it to explain how to setup an axon, built with BluePyOpt, containing an erbitrary number of sections. Afterward to simulate an actual cerebellar granule cell model with an axon composed by over 300 sections.",
+        "experience": ["power"],
+        "access": ["hpc"],
+        "disabled": false,
+        "picture": {
+          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/Granule_2019.png"
+        },
+        "next": "sc_form",
+        "files": [
+          {
+            "file": "ca27b360-28c8-4655-bb67-aea88b0903c1",
+            "file_prod": "831c846e-1d47-4903-ac53-83806c04be40",
+            "entryname": "Cerebellar granule cell custom axon and simulation",
+            "extension": ".ipynb",
+            "appid": 175,
+            "contenttype": "application/x-ipynb+json",
+            "justcopy": false,
+            "initial": true
+          },
+          {
+            "file": "ac517c7a-f941-4310-8f77-edd4523fe64c",
```

Comment:
Hi @SteMasoli please I would suggest instead of copy these mod/asc files like that I think it's way more efficient to create a zip file (or tar) of these 15 files that are not the notebook and then in the notebook uncompress it. Working with the Collab storage is slow so the less calls we do, the better

Reply:
Hi @antonelepfl .
I have pushed a new commit with a zip file for all the file used in the usecase.